docs/services/data/configure-data-query.md

```diff
@@ -0,0 +1,113 @@
+   Provide your organization's `org-id` from step 2, and a desired new password for your database user.
+
+   ```sh {class="line-numbers linkable-line-numbers"}
+   viam data database configure --org-id=<YOUR-ORGANIZATION-ID> --password=<NEW-DBUSER-PASSWORD>
+   ```
+
+   This command returns the `user` name you can use to connect to the Viam organization's MongoDB Atlas instance.
+
+1. Determine the hostname for your organization's MongoDB Atlas Data Federation instance.
+   Provide your organization's `org-id` from step 2:
+
+   ```sh {class="line-numbers linkable-line-numbers"}
+   viam data database hostname --org-id=<YOUR-ORGANIZATION-ID>
+   ```
+
+   This command returns the `hostname` (including database name) you can use to connect to your data store on the Viam organization's MongoDB Atlas instance.
+
+For more information, see the documentation for the [Viam CLI `database` command](/manage/cli/#data).
+
+## Query data
+
+Once you have [configured data query](#configure-data-query), you can directly query synced tabular data using MQL or SQL.
+
+You can use any client that is capable of connecting to MongoDB Atlas instances, including [the `mongosh` shell](https://www.mongodb.com/docs/mongodb-shell/), [MongoDB Compass](https://www.mongodb.com/docs/compass/current/), and many third-party tools.
+Use the `hostname` and `user` returned from the setup above to connect from your desired client to the MongoDB Atlas instance.
+
+For example, to connect to your Viam organization's MongoDB Atlas instance and query data using the `mongosh` shell:
+
+1. If you haven't already, [download the `mongosh` shell](https://www.mongodb.com/try/download/shell).
+   See the [`mongosh` documentation](https://www.mongodb.com/docs/mongodb-shell/) for more information.
+
+1. Run the following command to connect to the Viam organization's MongoDB Atlas instance from `mongosh`:
+
+   ```sh {class="line-numbers linkable-line-numbers"}
+   mongosh "mongodb+srv://<YOUR-DB-HOSTNAME>" --apiVersion 1 --username <YOUR-DB-USER>
+   ```
+
+   Where:
+
+   - `<YOUR-DB-HOSTNAME>` is your organization's assigned MongoDB Atlas instance hostname (including database name), as returned from `viam data database hostname` above.
+   - `<YOUR-DB-USER>` is your organization's database user for that Atlas instance, as returned from `viam data database configure` above.
+
+1. Once connected, you can run [MQL](https://www.mongodb.com/docs/manual/tutorial/query-documents/) or SQL to query captured data directly. For example:
+
+   - The following MQL query searches the `sensorData` database and `readings` collection, and returns sensor readings from an ultrasonic sensor on a specific `robot_id` where the recorded `distance` measurement is greater than `.2` meters:
+
+     ```sql {class="line-numbers linkable-line-numbers"}
```

Comment:
is the sql here just for markdown? should it be mql?

Reply:
Right on, I've just now [added](https://github.com/viamrobotics/docs/pull/2221) support for MQL syntax highlighting to our docs! 🎉

Reply:
Awesome!!